pkg/druid.go

```diff
@@ -726,6 +726,27 @@ func (ds *druidDatasource) prepareResponse(resp *druidResponse, settings map[str
 	response := backend.DataResponse{}
 	frame := data.NewFrame("response")
 	hideEmptyColumns, _ := settings["hideEmptyColumns"].(bool)
+	format, ok := settings["format"]
+	if !ok {
+		format = "long"
+	} else {
+		format = format.(string)
+	}
+	if format == "log" {
+		for ic, c := range resp.Columns {
+			var ff []string
+			ff = make([]string, 0)
+			if c.Type == "string" && c.Name == "message" {
+				for _, r := range resp.Rows {
+					if r[ic] == nil {
+						r[ic] = ""
+					}
+					ff = append(ff, r[ic].(string))
+				}
+				frame.Fields = append(frame.Fields, data.NewField("____message", nil, ff))
```

Comment:
(nit) would be nice to have a comment documenting the parameters so it's easier understood what `nil` is.

Reply:
oh, i actually don't know either. i'm just copying from what happens below. it's a good idea though and probably worth trying to understand it myself.

Reply:
oh, that's for the labels: https://github.com/grafana/grafana-plugin-sdk-go/blob/master/data/field.go#L49